css/responsive/desktop.styl

```diff
@@ -1,5 +1,15 @@
 // ******************************************************
 
+// FORMS
+
+input
```

Comment:
why not use a class? I think this is a bad practice, the rule's scope is way too general.